django/contrib/gis/db/backends/postgis/const.py

```diff
@@ -41,3 +43,19 @@
     'f': 4,  # Float
     'd': 8,  # Double
 }
+
+# Pixel type specifies type of pixel values in a band. Storage flag specifies
+# whether the band data is stored as part of the datum or is to be found on
+# the server's filesystem.
+
+# There are currently 11 supported pixel value types, so 4 bits are enough
+# to account for all. We'll reserve the upper 4 bits for generic flags and
+# define upmost as storage flag:
+
+BANDTYPE_FLAGS_MASK = 0xF0
+BANDTYPE_PIXTYPE_MASK = 0x0F
+
+BANDTYPE_FLAG_OFFDB = 1 << 7
+BANDTYPE_FLAG_HASNODATA = 1 << 6
+BANDTYPE_FLAG_ISNODATA = 1 << 5
+BANDTYPE_FLAG_RESERVED3 = 1 << 4
```

Comment:
`BANDTYPE_FLAG_RESERVED3`, `BANDTYPE_FLAG_ISNODATA` and `BANDTYPE_FLAG_OFFDB` are unused. Can we remove them?